### ISSUE TITLE
Support multiple tenants in route creation/editing/viewing

### ISSUE THREAD

**Issue by gr4per** [OpusCapita/routing#52]
This is driven by permissions and user can have the required permissions on different tenants.
Admin user usually has permission on all tenants.

Proposal is to evaluate req.opuscapita.getUserTenants() for the editor and back end in order to limit the choices for tenant via the assigned user2role + permission.

**Comment by kwierchris** [comment]
https://github.com/OpusCapita/bouncer#usage-example

**Comment by StefanDywersant** [comment]
It is already being done on server side:
https://github.com/OpusCapita/routing/blob/develop/src/server/routes/index.js#L314
https://github.com/OpusCapita/routing/blob/develop/src/server/api/route.js#L31

**Comment by nczeranka** [comment]
Please consider that ...getTenantId() is always part of ...getUserTenants()
The goal is that you can created Routes for other tenants, as long as getUserTenants allows you to do so.

**Comment by gr4per** [comment]
yup, so server side is clear with bouncer API, do we have a solution on how to support this on client side, too ?

**Comment by kwierchris** [comment]
I had no time yet to add a similar feature to the client side. We currently can only check in the client side bouncer if someone has permissions on a resource.
